ui/src/adapters/api/credentials.ts

```diff
@@ -125,7 +135,7 @@ export async function getCredentials({
 
 export type CreateCredential = {
   credentialSchema: string;
-  credentialSubject: Json;
+  credentialSubject: JsonLiteral | ObjectAttributePayload;
```

Comment:
Why this is param is not a `Json` anymore?

Reply:
I changed it to be explicit about the fact that `Json` includes arrays and we do not yet support list attributes, but it can also be kept as `Json` since `JsonLiteral | ObjectAttributePayload` is a subset of `Json`. Changed back to `Json`.